cmd/d8s/delete.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"log"
+	"net/http"
+
+	"github.com/google/go-containerregistry/authn"
+	"github.com/google/go-containerregistry/name"
+	"github.com/google/go-containerregistry/v1/remote"
+	"github.com/google/subcommands"
+)
+
+type deleteCmd struct{}
+
+func (*deleteCmd) Name() string             { return "delete" }
+func (*deleteCmd) Synopsis() string         { return "Deletes a reference from its registry" }
+func (*deleteCmd) Usage() string            { return "delete <image>" }
+func (*deleteCmd) SetFlags(f *flag.FlagSet) {}
```

Comment:
Shouldn't we parse `ref` here?

I think we should adopt standard terminology for `<tag>` vs. `<digest>` vs. `<reference>`.

Reply:
We should also think about whether we can share parsing logic for these.

Reply:
I think `SetFlags` is only intended as a helper to parse flags, not args. I admit that I don't know though.

I've updated the usages to specify `<reference>` or `<tag>`